fares-data-build-tool/create-data
# feat: CFD-282 hybrid period netex

# Description

-   To allow users of the site to create hybrid period ticket netex.

# Testing instructions

-   Make use of the dev repo's new matching JSON file for a hybrid period ticket, and observe the netex created (for SP to confirm if its ok).

# Type of change


# Checklist:
